packages/core/examples/colorbar/index.ts

```diff
@@ -0,0 +1,728 @@
+const bottomLeftContainer = document.createElement('div');
+const bottomRightContainer = document.createElement('div');
+
+const containers = [
+  rightTopContainer,
+  rightBottomContainer,
+  bottomLeftContainer,
+  bottomRightContainer,
+];
+
+const info = document.createElement('div');
+content.appendChild(info);
+
+const addInstruction = (instruction) => {
+  const node = document.createElement('p');
+  node.innerText = instruction;
+  info.appendChild(node);
+};
+
+addInstruction('- Select different colormaps');
+addInstruction('- Click and drag on viewport to change VOI');
+addInstruction('- Click and drag at the color bar to change VOI');
+
+const colorBarSize = { shortSide: 20, longSide: 250 };
+
+// const ctColorBar = new ViewportColorBar({
+//   id: 'ctColorBar',
+//   viewportId: volumeViewportId,
+//   renderingEngineId,
+//   colormaps,
+//   activeColormapName: 'Grayscale',
+//   // voiRange: { min: 0.25, max: 0.75 },
+// });
+
+// const ptColorBar = new ViewportColorBar({
+//   id: 'ptColorBar',
+//   viewportId: volumeViewportId,
+//   renderingEngineId,
+//   colormaps,
+//   activeColormapName: currentPTColormapName,
+//   // voiRange: { min: 0.25, max: 0.75 },
+// });
+
+// const colorBars = [ctColorBar, ptColorBar */];
+
+// ctColorBar.rootElement.draggable = true;
+// ctColorBar.rootElement.style.cursor = 'move';
+
+// ptColorBar.rootElement.draggable = true;
+// ptColorBar.rootElement.style.cursor = 'move';
+
+Object.assign(bottomLeftContainer.style, {
+  position: 'absolute',
+  top: '100%',
+  left: '0px',
+  width: `${colorBarSize.longSide}px`,
+  height: `${colorBarSize.shortSide}px`,
+});
+
+Object.assign(bottomRightContainer.style, {
+  position: 'absolute',
+  top: '100%',
+  left: '50%',
+  width: `${colorBarSize.longSide}px`,
+  height: `${colorBarSize.shortSide}px`,
+});
+
+Object.assign(rightTopContainer.style, {
+  position: 'absolute',
+  top: '0px',
+  left: '100%',
+  width: `${colorBarSize.shortSide}px`,
+  height: `${colorBarSize.longSide}px`,
+});
+
+Object.assign(rightBottomContainer.style, {
+  position: 'absolute',
+  top: '50%',
+  left: '100%',
+  width: `${colorBarSize.shortSide}px`,
+  height: `${colorBarSize.longSide}px`,
+});
+
+// Change the container style when it has/hasn't a colorbar attached to it
+const containersMutationObserver = new MutationObserver((mutations) => {
+  mutations.forEach((mutation) => {
+    const container = mutation.target as HTMLElement;
+    const hasChildNodes = container.hasChildNodes();
+
+    Object.assign(container.style, {
+      display: hasChildNodes ? 'block' : 'none',
+      border: hasChildNodes ? 'solid 1px #555' : 'none',
+    });
+  });
+});
+
+containers.forEach((container) => {
+  const hasChildNodes = container.hasChildNodes();
+
+  Object.assign(container.style, {
+    boxSizing: 'border-box',
+    display: hasChildNodes ? 'block' : 'none',
+    cursor: 'initial',
+  });
+
+  // container.addEventListener('dragover', (evt) => evt.preventDefault());
```

Comment:
There are so many commented lines from here down, so I don't think this is ready for review